machine.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Hosts": [
```

Comment:
Remove please.

Reply:
alcor/config/machine.json file is removed.